### DISCUSSION TITLE
[Bug]: Pimcore X Bug

### DISCUSSION THREAD

**Discussion by kumar2112** [pimcore/pimcore#10065] [Q&A / Support]
### Expected behavior

Should Take On Login Page

### Actual behavior

![image](https://user-images.githubusercontent.com/12470739/129865815-df46ff0f-2ef4-470d-b84f-c7d1a598c9ef.png)

### Steps to reproduce

Steps 
1. Execute the command "COMPOSER_MEMORY_LIMIT=-1 composer update"
2. Browse the login page in browser throwing below error.

`Symfony\Component\Translation\MessageCatalogue::addCatalogue(): Argument #1 ($catalogue) must be of type Symfony\Component\Translation\MessageCatalogueInterface, string given, called in D:\xampp\htdocs\Pimcore_Panasonic\pimcore_panasonic\vendor\pimcore\pimcore\lib\Translation\Translator.php on line 256`

**Comment by kumar2112** [comment]
If i am trying to rebuild the classes it shows.

![image](https://user-images.githubusercontent.com/12470739/129865684-acdb19a4-dde5-4c4f-8320-a90ae30fcd21.png)

**Comment by brusch** [comment]
@kumar2112 This seems to be specific to your setup -> moving it over to Discussions ...
